BridgeBidder/Constraints/PositionProxy.cs

```diff
@@ -29,5 +29,20 @@ public override bool Conforms(Call call, PositionState ps)
 		{
 			return _constraint.Conforms(call, GetPosition(ps));
 		}
+
+		public string Describe(Call call, PositionState ps)
+		{
+			if (_constraint is IDescribeConstraint describeConstraint)
+			{
+				return $"{_relativePosition} {describeConstraint.Describe(call, GetPosition(ps))}";
```

Comment:
Minor nit, but I think this results in "Partner" instead of "partner" in the description. I'd prefer to have it be lowercase (though LHO and RHO should be fine).